package.nls.json

```diff
@@ -422,18 +422,25 @@
     "AWS.s3.downloadFile.progressTitle": "Downloading {0}...",
     "AWS.s3.downloadFile.saveButton": "Download",
     "AWS.s3.downloadFile.error.general": "Failed to download file {0}",
+    "AWS.s3.fileViewer.button.dismiss": "Don't show this again",
+    "AWS.s3.fileViewer.button.help": "Help",
+    "AWS.s3.fileViewer.button.cancelUpload": "Cancel upload",
+    "AWS.s3.fileViewer.button.overwrite": "Overwrite",
     "AWS.s3.promptTime.placeHolder": "Defaults to 15 minutes",
     "AWS.s3.promptTime.prompt": "Enter a valid time (in minutes) for URL: ${0}",
     "AWS.s3.fileViewer.message.tempCreation": "Temp folder for FileViewer created with location: {0}",
-    "AWS.s3.fileViewer.warning.noSize": "File size couldn't be determined. Continue with download?",
-    "AWS.s3.fileViewer.warning.4mb": "File size is more than 4MB. Continue with download?",
     "AWS.s3.fileViewer.message.fileKey": "Retrieving and displaying file {0}",
     "AWS.s3.fileViewer.message.fileLocation": "File to be opened is: {0}",
     "AWS.s3.fileViewer.message.sizeLimitCancellation": "Download cancelled",
     "AWS.s3.fileViewer.message.noSizeCancellation": "Download cancelled",
     "AWS.s3.fileViewer.error.download": "Error downloading file {0} from S3: {1}",
-    "AWS.s3.fileViewer.error.invalidSize": "Files over 50MB currently not supported for file display, please use the \"download as\" function",
+    "AWS.s3.fileViewer.error.invalidSize": "Files over 50MB currently not supported for file display, please use the \"Download as...\" function",
+    "AWS.s3.fileViewer.error.invalidUpload": "File is invalid to upload, file has changed in S3 since last cache download. Please compare your version with the one in S3. Then decide if you want to overwrite them or cancel this upload.",
```

Comment:
```suggestion
    "AWS.s3.fileViewer.error.invalidUpload": "File has changed in S3 since last cache download. Compare your version with the one in S3, then choose to overwrite it or cancel this upload.",
```